i3 has issues building when parallel builds are enabled.

(From https://aur.archlinux.org/packages/i3-git/#comment-739440)

When parallel builds are enabled, i3 build can fail with various errors. One such error:
[error.log](https://github.com/i3/i3/files/4480799/error.log)

Workaround is to add `-j1` to `make`.

I'm experiencing this with Manjaro linux running on Ryzen 3700X.

Hmm I get an error as well sometimes:

```
rm -f libi3.a
gcc -DHAVE_CONFIG_H -I. -I../../i3  -DSYSCONFDIR="\"/usr/local/etc\"" -I./parser -I../../i3/include -DSTRIPPED__FILE__=\"$(basename ../../i3/src/commands_parser.c)\"   -Wunused-value -Wunused-value -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include    -I/usr/include/yajl -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/pango-1.0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/startup-notification-1.0       -I/usr/include/yajl  -pthread  -g -O0 -Wall -MT src/i3-commands_parser.o -MD -MP -MF src/.deps/i3-commands_parser.Tpo -c -o src/i3-commands_parser.o `test -f 'src/commands_parser.c' || echo '../../i3/'`src/commands_parser.c
gcc -DHAVE_CONFIG_H -I. -I../../i3  -DSYSCONFDIR="\"/usr/local/etc\"" -I./parser -I../../i3/include -DSTRIPPED__FILE__=\"$(basename ../../i3/src/config_parser.c)\"   -Wunused-value -Wunused-value -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include    -I/usr/include/yajl -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/pango-1.0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/startup-notification-1.0       -I/usr/include/yajl  -pthread  -g -O0 -Wall -MT src/i3-config_parser.o -MD -MP -MF src/.deps/i3-config_parser.Tpo -c -o src/i3-config_parser.o `test -f 'src/config_parser.c' || echo '../../i3/'`src/config_parser.c
ar cru libi3.a libi3/libi3_a-dpi.o libi3/libi3_a-draw_util.o libi3/libi3_a-fake_configure_notify.o libi3/libi3_a-font.o libi3/libi3_a-format_placeholders.o libi3/libi3_a-g_utf8_make_valid.o libi3/libi3_a-get_colorpixel.o libi3/libi3_a-get_config_path.o libi3/libi3_a-get_exe_path.o libi3/libi3_a-get_mod_mask.o libi3/libi3_a-get_process_filename.o libi3/libi3_a-get_visualtype.o libi3/libi3_a-ipc_connect.o libi3/libi3_a-ipc_recv_message.o libi3/libi3_a-ipc_send_message.o libi3/libi3_a-is_debug_build.o libi3/libi3_a-mkdirp.o libi3/libi3_a-resolve_tilde.o libi3/libi3_a-root_atom_contents.o libi3/libi3_a-safewrappers.o libi3/libi3_a-string.o libi3/libi3_a-strndup.o libi3/libi3_a-ucs2_conversion.o 
gcc -Wunused-value -Wunused-value -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include    -I/usr/include/yajl -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/pango-1.0 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/startup-notification-1.0  -g -O0 -Wall   -o i3-config-wizard/i3-config-wizard i3-config-wizard/i3_config_wizard_i3_config_wizard-main.o ./libi3.a -lgobject-2.0 -lglib-2.0 -lxcb -lxcb-xkb -lxcb-xinerama -lxcb-randr -lxcb-shape -lxcb-util -lxcb -lxcb-xrm -lxcb -lyajl -lpangocairo-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lcairo -lstartup-notification-1 -lxcb-keysyms -lxcb -lxkbcommon-x11 -lxkbcommon -lrt -lev -lm 
gcc: error: ./libi3.a: No such file or directory
Makefile:1553: recipe for target 'i3-config-wizard/i3-config-wizard' failed
make[1]: *** [i3-config-wizard/i3-config-wizard] Error 1
make[1]: *** Waiting for unfinished jobs....
ar: `u' modifier ignored since `D' is the default (see `U')
ranlib libi3.a
mv -f src/.deps/i3-commands_parser.Tpo src/.deps/i3-commands_parser.Po
mv -f src/.deps/i3-config_parser.Tpo src/.deps/i3-config_parser.Po
make[1]: Leaving directory '/home/orestis/KTH/soft-paraphrase/build/i3/build'
Makefile:1284: recipe for target 'all' failed
make: *** [all] Error 2
+ exit 2
```

--- a/Makefile.am
+++ b/Makefile.am
@@ -442,7 +442,8 @@ i3_config_wizard_i3_config_wizard_SOURCES = \
 	i3-config-wizard/xcb.h
 
 i3_config_wizard_i3_config_wizard_DEPENDENCIES = \
-	$(config_parser_SOURCES)
+	$(config_parser_SOURCES) \
+	$(top_builddir)/libi3.a
 
 test_inject_randr15_CPPFLAGS = \
 	$(AM_CPPFLAGS)

